spec/factories/posts.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+
+  factory :post do
+    title 'Lorem'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.